projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/ai-image/components/ai-image-modal.tsx

```diff
@@ -138,13 +138,24 @@ export default function AiImageModal( {
 					<div className="ai-image-modal__content">
 						<div className="ai-image-modal__user-prompt">
 							<div className="ai-image-modal__user-prompt-textarea">
-								<textarea
-									disabled={ disableInstructions }
-									maxLength={ 1000 }
-									rows={ 2 }
-									onChange={ handleUserPromptChange }
-									placeholder={ instructionsPlaceholder }
-								></textarea>
+								<KeyboardShortcuts
+									bindGlobal
+									shortcuts={ {
+										enter: () => {
+											if ( ! disableGenerateButton ) {
```

Comment:
Unrelated to this PR:
`disableGenerateButton` reads like a function. I would suggest changing it to `generateButtonDisabled`. Same for the other variables.

Reply:
Pressing Enter once on a page with no content and no instructions do not trigger a generation, but it adds a newline, and then subsequent enters do trigger the generation.

My suggestion here is to consider the button disabled not only if there is no content, but also if there is only whitespace.

Reply:
Addressed both your feedbacks, renaming some vars and also trimming the content of the user prompt so remove extra spaces on it, so a field with only new lines can't be submitted.